### DISCUSSION TITLE
通过函数或远程方法获取下拉列表值的实现

### DISCUSSION THREAD

**Discussion by scottwansz** [dream-num/Luckysheet#1122] [Ideas]
因工作的需要，现在实现单元格从下拉列表取值时，可以使用函数或远程方法，工作进行中，估计需要一些时间，请各位高人指点。
具体实现思路可见：【腾讯文档】Luckysheet-远程数据
https://docs.qq.com/doc/DV1h4VEpOVHBha1JP

**Comment by keepanote** [comment]
有实现的方法吗？

**Reply by scottwansz** [reply]
sorry, my project terminated. need close this.
